tools/bookrunner/librustdoc/build.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Build script that allows us to build this dependency without bootstrap script.
+pub fn main() {
+    // Hard code nightly configuration to build librustdoc.
+    println!("cargo:rustc-env=DOC_RUST_LANG_ORG_CHANNEL=nightly");
```

Comment:
the ci script is still setting this right?

also, should that change to `cargo build` too?

Reply:
Indeed. We should update the CI script.

Not sure what you mean in the second question. Can you please clarify?